happybase/batch.py

```diff
@@ -45,7 +45,7 @@ def _reset_mutations(self):
 
     def send(self):
         """Send the batch to the server."""
-        bms = [BatchMutation(row, m) for row, m in self._mutations.iteritems()]
+        bms = [BatchMutation(row, m) for row, m in self._mutations.items()]
```

Comment:
Yes.  I thought about creating a separate utility function just for iterating over items:

    if sys.version_info >= (3,):
        def iteritems(d):
            return d.items()
    else:
        def iteritems(d):
            return d.iteritems()

PEP 469 [recommends this approach](http://legacy.python.org/dev/peps/pep-0469/#migrating-to-the-common-subset-of-python-2-and-3), though also mentions that `six` and `future.utils` provide the same functionality.